spec/requests/splash_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe 'Splash', type: :request do
+  describe 'GET /home' do
+    before(:example) { get unauthenticated_root_path }
+
+    it 'checks if action returns correct response status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'checks if action rendered a correct template' do
+      expect(response).to render_template('home')
+    end
+
+    it 'checks if correct title is shown' do
+      expect(response.body).to include('ManageMyMoney')
+      expect(response.body).to include('SIGN UP')
+      expect(response.body).to include('SIGN IN')
+    end
```

Comment:
- [ ] This test which was passing 🟢 is now failing after your latest changes. Please try to make it green ✔️ 
![image](https://user-images.githubusercontent.com/46839548/227700564-500fac35-aa15-4214-a3e3-6f00b3e88bad.png)